sdbuild/scripts/create_bsp.sh

```diff
@@ -6,6 +6,13 @@ set -e
 board=$1
 template=$2
 
+# If $BSP is a URL, fetch it!
+if [[ "$BSP" == *"://"* ]] ; then
```

Comment:
Can you move this if block inside the `if [ ! -z "$BSP" ]` block? It would not be executed anyway if BSP is empty, so it makes things more readable.

Reply:
My original thinking was to have the block on the outside as a fallback, but yeah, there's clearly no point.  I'll change this.